ci/aws/README.md

```diff
@@ -34,7 +34,7 @@ terraform apply -auto-approve \
 Should the script either fail or configued not to cleanup, destruction of the
```

Comment:
Oops, another ansible artifact. I guess this should say something like:
Please always destroy your environment after completion...